app/views/policies/index.html.erb

```diff
@@ -3,8 +3,8 @@
 <h1>Policies</h1>
 
 <p>
-  <%= link_to "New policy area", new_policy_path, class: 'btn btn-primary' %>
-  <%= link_to "New policy programme", new_policy_path(programme: true), class: 'btn btn-primary' %>
+  <%= link_to "New policy", new_policy_path, class: 'btn btn-primary' %>
+  <%= link_to "New sub-policy", new_policy_path(programme: true), class: 'btn btn-primary' %>
```

Comment:
It would be nice to tidy up this parameter name too (perhaps to `sub_policy: true`).